src/models.ts

```diff
@@ -27,21 +31,27 @@ export class QueryResult {
 
     static async run(client : QueryApi, query : string) : Promise<QueryResult> {
         const result = new QueryResult()
-        const currentTableId = -1
+        let currentTableId = -1
         let currentTableResult : TableResult
         return new Promise((resolve, reject) => {
             client.queryRows(query, {
                 next(row : string[], tableMeta : FluxTableMetaData) {
                     const idColumn = tableMeta.column('table')
                     const idIndex = tableMeta.columns.indexOf(idColumn)
-                    if (currentTableId !== parseInt(row[idIndex])) {
+                    const rowTableId = parseInt(row[idIndex])
+                    if (currentTableId !== rowTableId) {
+                        console.log(`Old id is ${currentTableId} and new id is ${parseInt(row[idIndex])}`)
```

Comment:
I guess console.log should be removed from the production code, or if required, you can put some sort of ENV check if not production then print?
I am not an expert in this area so need some clarification.

Reply:
I did remove this, and probably ought to add a lint rule about it. There are a few cases where I'd like `console.log` output when people might file a bug, so I don't want to completely kill it, but I want to be more intentional about adding `console.log`